examples/rcbilling-demo/src/tests/main.test.ts

```diff
@@ -79,6 +118,19 @@ test.describe("Main", () => {
   });
 });
 
+async function performPurchase(page: Page, card: Locator, userId: string) {
+  // Perform purchase
+  const cardButton = card.getByRole("button");
+  await cardButton.click();
+
+  await enterEmailAndContinue(page, userId);
+  await enterCreditCardDetailsAndContinue(page);
+
+  // Confirm success page has shown.
+  const successText = page.getByText("Purchase successful");
+  await expect(successText).toBeVisible({ timeout: 10000 });
```

Comment:
I was getting some failures with the standard (5000ms) so I increased it to 10000... Maybe we can optimize it in the future when it becomes a problem.

Reply:
Does this only happen for non-subscription purchases?

Reply:
Yeah those were the failures... Not sure if the backend was taking longer for some reason for those 🤔

Reply:
That's weird, it shouldn't take longer than subscriptions.

Reply:
Right... FWIW, it's taking longer to display the `Purchase successful` text. I'm able to reproduce quite easily locally (subscriptions works almost always and consumables fail quite often).

From running manually (I've done 3 subscriptions and 3 consumable purchases), subscriptions take ~3 poll attempts while consumables ~4-5.

Reply:
I'll take a look, thanks for point it out 👍